apps/aecore/src/aecore.app.src

```diff
@@ -12,7 +12,9 @@
     rocksdb,
     aeutils
    ]},
-  {env, []},
+  {env, [
+        {exometer_predefined, {script, "$PRIV_DIR/exometer_predefined.script"}}
```

Comment:
I see `$PRIV_DIR` is achieved by app `setup` as commented [here](https://github.com/Feuerlabs/exometer_core/issues/58#issuecomment-169685186).

Reply:
Yes, `setup` has quite a few tricks regarding configuration. Exometer uses it among other things to find exometer-related configs in other application environments.